govc/flags/client.go

```diff
@@ -428,11 +428,15 @@ func (flag *ClientFlag) newClient() (*vim25.Client, error) {
 	m := session.NewManager(c)
 	u := flag.url.User
 
-	if u.Username() == "" {
-		// Assume we are running on an ESX or Workstation host if no username is provided
-		u, err = flag.localTicket(ctx, m)
-		if err != nil {
-			return nil, err
+	if u.Username() == "" && !c.IsVC() {
+		// If no username is provided, try to acquire a local ticket.
+		// When invoked remotely, ESX returns an InvalidRequestFault (VC returns NotSupportedFault).
```

Comment:
this parenthetical is slightly confusing considering the conditional on line 431 excludes this path for VC.

elaborate or remove?

Reply:
Removed.  Thanks for reviewing Brad!